.github/workflows/auto_tag.yml

```
@@ -19,7 +19,7 @@ on:
 
 jobs:
   preparation:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Let's never experience an issue because Github Action OS defaults change :sweat_smile: 
```suggestion
    runs-on: ubuntu-24.04
```

---

I prefer to experience issues and be on the cutting edge, otherwise, we will be postponing upgrades all the time 🤣

---

I would prefer a stable deployment procedure, not cutting edge. But that's your domain, your call.